packages/chakra-ui/src/Slider/index.js

```diff
@@ -306,14 +310,17 @@ const Slider = forwardRef(
           role="presentation"
           tabIndex="-1"
           onMouseDown={handleMouseDown}
+          onTouchStart={handleMouseDown}
           onMouseLeave={handleMouseUp}
+          onTouchEnd={handleMouseUp}
           onBlur={event => {
             handleMouseUp();
             onBlur && onBlur(event);
           }}
           py={3}
           aria-disabled={isDisabled}
           ref={ref}
+          css={{ touchAction: "none" }}
```

Comment:
`css={{ touchAction: "none" }}` was added...
1. To prevent scrolling while moving the slider's thumb
2. To not cause the `"react-dom.development.js:1506 [Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/features/5093566007214080"` bug on chrome.

Reply:
I see what you mean. Thanks for looking into this.

I'll test it on my mobile devices and merge it.

Reply:
You're welcome :) Btw great work on chakra!